commands/pull.js

```diff
@@ -188,10 +188,10 @@ TARGET will be created locally if it's a database name or remotely if it's a ful
 Examples:
 
   # pull Heroku DB named postgresql-swimmingly-100 into local DB mylocaldb
-  $ heroku pg:push postgresql-swimmingly-100 mylocaldb
+  $ heroku pg:pull HEROKU_POSTGRESQL_MAGENTA mylocaldb --app sushi
```

Comment:
I *think* we are trying to get people more familiar with addon haiku name than color, so maaaybe the example can stay using haiku name. I'm not super positive about it, though.
Also, if we're going to change here, we should change the example of `pg:push` as well.